dev/com.ibm.ws.microprofile.config.1.1_fat/test-applications/cdiConfig.jar/src/com/ibm/ws/microprofile/appConfig/cdi/beans/ConfigFieldInjectionInterface.java

```diff
@@ -48,6 +48,8 @@
 
     public URL getOPTIONAL_MISSING_KEY();
 
+    public String getMISSING_KEY();
+
     public Pizza getPIZZA_KEY();
```

Comment:
Not sure that this interface is actually referenced anywhere apart from the bean that implements it. Maybe it could be taken out?

Reply:
Yes, you're right, only ConfigFieldInjectionBean.java implements this interface. I'd be happy to take it out. This would also resolve this comment: https://github.com/OpenLiberty/open-liberty/pull/14375#discussion_r504081213